examples/Getting_Started/Getting-started-with-Merlin-Systems.ipynb

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    triton_chain = TransformWorkflow(workflow) &gt;&gt; PredictTensorflow(model)
</pre><p>Could we call this variable something like "serving_operators"? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/systems/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>